src/Compilers/Core/Portable/CodeGen/PrivateImplementationDetails.cs

```diff
@@ -132,6 +135,12 @@ private static string GetClassName(string moduleName, int submissionSlotIndex, b
             return name;
         }
 
+        internal static string GetInlineArrayTypeName(int arrayLength)
+        {
+            Debug.Assert(arrayLength > 0);
+            return $"$InlineArray{arrayLength}";
```

Comment:
nit: the other nested types we add in private implementation details have a different naming convention. Consider aligning. Here's one that was recently added:
```
        public string Name => _alignment == 1 ?
            $"__StaticArrayInitTypeSize={_size}" :
            $"__StaticArrayInitTypeSize={_size}_Align={_alignment}";
```

Reply:
Changed to `__InlineArrayN<T>` but perhaps `__InlineArrayTypeSize=N<T>` might be better, or at least more consistent. Feel free to suggest specific names.